common/lib/capa/capa/responsetypes.py

```diff
@@ -1083,16 +1255,80 @@ def late_transforms(self, problem):
         self.do_shuffle(self.xml, problem)
         self.do_answer_pool(self.xml, problem)
 
-    def get_score(self, student_answers):
+    def grade_via_points(self, **kwargs):
```

Comment:
The code you're pointing to in the old version is in the OptionResponse class, which is the dropdown problem type. The code you've referenced above is in MultipleChoiceResponse instead. The equivalent line in the old code is here:
https://github.com/edx/edx-platform/pull/8724/files#diff-38722d7e328c77e747d36e85e7ab2c0fR1257

You probably got confused because both dropdown and MC had "points"-style grading.

To my knowledge, the only code that is different when you compare my new new responsetypes.py file to my old one is in the OptionResponse class, because I stripped out the partial credit there so I could fix it in a different PR. The MultipleChoiceResponse class should be identical to the old one.

Does that help?